eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -92,6 +93,14 @@ case class NodeParams(keyManager: KeyManager,
 
 object NodeParams {
 
+  val defaultFeatures = ConfigFactory.parseMap(Map(
+    s"features.${InitialRoutingSync.rfcName}" -> "optional",
+    s"features.${OptionDataLossProtect.rfcName}" -> "optional",
+    s"features.${ChannelRangeQueries.rfcName}" -> "optional",
+    s"features.${ChannelRangeQueriesExtended.rfcName}" -> "optional",
+    s"features.${VariableLengthOnion.rfcName}" -> "optional"
+  ).asJava)
+
```

Comment:
I don't understand why we need to hard-code this ?
Instead we could define application features in the relevant `application.conf` (eclair-node, eclair-mobile, phoenix), and use a specific configuration file for testing.

Reply:
Because any configuration that we put in `application.conf` will be merged with the user defined configuration in `eclair.conf`. Sure we can have a different `application.conf` for eclair-mobile/phoenix but for eclair-node we still need to make sure that we don't merge the user defined features with our default. 

For example a user placing the following config block in its `eclair.conf` expects to activate only `option_support_large_channel` and no other options, this is not the case at the moment because we merge with our default features and result in a quite different feature set.
```
  features {
    option_support_large_channel = optional
  }
```